backend/dissemination/api/api_v1_0_3/create_views.sql

```diff
@@ -0,0 +1,331 @@
+        pass.passthrough_id,
+        pass.passthrough_name
+    from
+        dissemination_general as gen,
+        dissemination_passthrough as pass
+    where
+        (gen.report_id = pass.report_id
+        and
+        gen.is_public = true)
+        or (gen.is_public = false and has_tribal_data_access())
+    order by pass.id
+;
+
+
+---------------------------------------
+-- general
+---------------------------------------
+create view api_v1_0_3.general as
+    select
+        -- every table starts with report_id, UEI, and year
+        gen.report_id,
+        gen.auditee_uei,
+        gen.audit_year,
+        ---
+        gen.auditee_certify_name,
+        gen.auditee_certify_title,
+        gen.auditee_contact_name,
+        gen.auditee_email,
+        gen.auditee_name,
+        gen.auditee_phone,
+        gen.auditee_contact_title,
+        gen.auditee_address_line_1,
+        gen.auditee_city,
+        gen.auditee_state,
+        gen.auditee_ein,
+        gen.auditee_zip,
+        -- auditor
+        gen.auditor_phone,
+        gen.auditor_state,
+        gen.auditor_city,
+        gen.auditor_contact_title,
+        gen.auditor_address_line_1,
+        gen.auditor_zip,
+        gen.auditor_country,
+        gen.auditor_contact_name,
+        gen.auditor_email,
+        gen.auditor_firm_name,
+        gen.auditor_foreign_address,
+        gen.auditor_ein,
+        -- agency
+        gen.cognizant_agency,
+        gen.oversight_agency,
+        -- dates
+        gen.date_created,
+        gen.ready_for_certification_date,
+        gen.auditor_certified_date,
+        gen.auditee_certified_date,
+        gen.submitted_date,
+        gen.fac_accepted_date,
+        gen.fy_end_date,
+        gen.fy_start_date,
+        gen.audit_type,
+        gen.gaap_results,
+        gen.sp_framework_basis,
+        gen.is_sp_framework_required,
+        gen.sp_framework_opinions,
+        gen.is_going_concern_included,
+        gen.is_internal_control_deficiency_disclosed,
+        gen.is_internal_control_material_weakness_disclosed,
+        gen.is_material_noncompliance_disclosed,
+        gen.dollar_threshold,
+        gen.is_low_risk_auditee,
+        gen.agencies_with_prior_findings,
+        gen.entity_type,
+        gen.number_months,
+        gen.audit_period_covered,
+        gen.total_amount_expended,
+        gen.type_audit_code,
+        gen.is_public,
+        gen.data_source,
+        gen.is_aicpa_audit_guide_included,
+        gen.is_additional_ueis,
+        CASE
+        WHEN aud.general_information ->> 'multiple_eins_covered' = 'true' THEN 'Yes'
+        WHEN aud.general_information ->> 'multiple_eins_covered' = 'false' THEN 'No'
+        ELSE 'null'
+    END AS is_multiple_eins,
+    CASE
+        WHEN aud.general_information ->> 'secondary_auditors_exist' = 'true' THEN 'Yes'
+        WHEN aud.general_information ->> 'secondary_auditors_exist' = 'false' THEN 'No'
+        ELSE 'null'
+    END AS is_secondary_auditors
+    from
```

Comment:
I haven't done cases before, but just checking on the indentation. I would have expected something like this:

```suggestion
        gen.data_source,
        gen.is_aicpa_audit_guide_included,
        gen.is_additional_ueis,
        CASE
            WHEN aud.general_information ->> 'multiple_eins_covered' = 'true' THEN 'Yes'
            WHEN aud.general_information ->> 'multiple_eins_covered' = 'false' THEN 'No'
            ELSE 'null'
        END AS is_multiple_eins,
        CASE
            WHEN aud.general_information ->> 'secondary_auditors_exist' = 'true' THEN 'Yes'
            WHEN aud.general_information ->> 'secondary_auditors_exist' = 'false' THEN 'No'
            ELSE 'null'
        END AS is_secondary_auditors
    from
```

Reply:
Made the changes in latest commit.